app/resources/views/pages/dominion/status.blade.php

```diff
@@ -191,52 +191,15 @@
                     </div>
                 </div>
             @endif
+        </div>
 
-            @if ($selectedDominion->pack !== null)
-                <div class="box">
-                    <div class="box-header with-border">
-                        <h3 class="box-title">Pack</h3>
-                    </div>
-                    <div class="box-body">
-                        <p>You are in pack <em>{{$selectedDominion->pack->name}}</em> with:</p>
-                        <ul>
-                            @foreach ($selectedDominion->pack->dominions as $dominion)
-                                <li>
-                                    @if ($dominion->ruler_name === $dominion->name)
-                                        <strong>{{ $dominion->name }}</strong>
-                                    @else
-                                        {{ $dominion->ruler_name }} of <strong>{{ $dominion->name }}</strong>
-                                    @endif
-
-                                    @if($dominion->ruler_name !== $dominion->user->display_name)
-                                        ({{ $dominion->user->display_name }})
-                                    @endif
-                                </li>
-                            @endforeach
-                        </ul>
-                        <p>
-                            Slots used: {{ $selectedDominion->pack->dominions->count() }} / {{ $selectedDominion->pack->size }}.
-                            @if ($selectedDominion->pack->isFull())
-                                (full)
-                            @elseif ($selectedDominion->pack->isClosed())
-                                (closed)
-                            @endif
-                        </p>
-                        @if (!$selectedDominion->pack->isFull() && !$selectedDominion->pack->isClosed())
-                            <p>Your pack will automatically close on <strong>{{ $selectedDominion->pack->getClosingDate() }}</strong> to make space for random players in your realm.</p>
-                            @if ($selectedDominion->pack->creator_dominion_id === $selectedDominion->id)
-                                <p>
-                                    <form action="{{ route('dominion.misc.close-pack') }}" method="post">
-                                        @csrf
-                                        <button type="submit" class="btn btn-link" style="padding: 0;">Close Pack Now</button>
-                                    </form>
-                                </p>
-                            @endif
-                        @endif
-                    </div>
+        @if ($selectedDominion->realm->message)
+            <div class="col-sm-12 col-md-9">
+                <div class="panel panel-warning">
+                    <div class="panel-heading"><b>Message of the Day:</b> {{ $selectedDominion->realm->message }}</div>
```

Comment:
Can we show the datetime when the message was last updated? And possibly by who (current monarch)? Not sure about the last one